pkg/system/filesys_windows.go

```diff
@@ -65,6 +86,34 @@ func MkdirAll(path string, perm os.FileMode) error {
 	return nil
 }
 
+// mkdirWithACL creates a new directory with the specified name and permission bits.
```

Comment:
Update the comment to reflect that it grants full access to administrators and system.